src/lib.rs

```diff
@@ -9,12 +9,14 @@ pub use error::*;
 
 #[doc(hidden)]
 pub use patches::PatchesRef;
+pub use app_desc::*;
 
 mod alloc;
 mod error;
 mod panic;
 mod patches;
 mod start;
+mod app_desc;
```

Comment:
Should be in an alphabetical order.

Reply:
Ha. Usually I'm the one sensitive to alphabetical ordering :)